client/dashboard/sites/backups/index.tsx

```diff
@@ -50,6 +59,40 @@ export function SiteBackupsCallout( {
 	);
 }
 
+function Backups( { site }: { site: Site } ) {
+	const [ view, setView ] = useState< View >( {
+		type: 'table',
+		fields: [ 'date', 'action', 'user' ],
+	} );
+
+	const { data: activityLog, isLoading: isLoadingActivityLog } = useQuery(
+		siteLastRewindableActivityLogEntriesQuery( site.ID )
+	);
+
+	const rawData = activityLog || [];
```

Comment:
[nit] we can combine it above

```ts
const { data: activityLog = []
```

Reply:
Thanks for the suggestion. I've added that and dropped the unneeded `rawData` variable.